cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1390,13 +1392,29 @@ void zookeeperPodSetDescription(StrimziPodSet podSet) {
         }
 
         Future<ReconciliationState> zkPodDisruptionBudget() {
+            if (pfa.hasPodDisruptionBudgetV1()) {
```

Comment:
Should this be negative? `if (!pfa.hasPodDisruptionBudgetV1()) {`?

Reply:
yes, good catch!